.circleci/functional_tests.sh

```diff
@@ -16,6 +16,8 @@ curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-
 sudo apt-get update
 sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev git
 export PATH="~/.pyenv/bin:$PATH"
+eval "$(pyenv init -)"
```

Comment:
Do we need to initialize pyenv? Since we are using `venv` when running the tests?